src/main/groovy/com/rundeck/plugins/azure/plugin/AzureResourceModelSource.groovy

```diff
@@ -42,14 +42,20 @@ class AzureResourceModelSource  implements ResourceModelSource {
         String key=configuration.getProperty(AzureResourceModelSourceFactory.KEY)
         String pfxCertificatePath=configuration.getProperty(AzureResourceModelSourceFactory.PFX_CERTIFICATE_PATH)
         String pfxCertificatePassword=configuration.getProperty(AzureResourceModelSourceFactory.PFX_CERTIFICATE_PASSWORD)
-        String resourceGroup=configuration.getProperty(AzureResourceModelSourceFactory.RESOURCE_GROUP)
```

Comment:
we cannot remove an old attribute because it will break existing configurations. You can use the same name but change the description

Reply:
It kept the old attribute, I just changed the variable name from `RESOURCE_GROUP` to `RESOURCE_GROUPS` but the value remains the same (`"resourceGroup"`) so serialization doesn't change at all

Reply:
perfect